app/views/admin/users/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="userInfo">
+  <p><%= current_user.name  %></p>
+  <p><%= current_user.address  %></p>
+  <p><%= current_user.city  %></p>
+  <p><%= current_user.state  %></p>
+  <p><%= current_user.zip  %></p>
+  <p><%= current_user.email  %></p>
+  <p><%= current_user.role  %></p>
+  <%= link_to 'My Orders', profile_orders_path unless current_user.orders.empty? %>
```

Comment:
I think this may be getting ahead to a future user story, but since this is here, I'd change all instances of `current_user` to `@user` (since current_user in this case will be the admin, so you see the admin's information instead of that particular user's)

Reply:
I like your logic. Updated!